src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -0,0 +1,852 @@
+      } else {
+        rowCells.push(
+          <div
+            class={`${DATA_TABLE_CLASSES.CELL} ${DATA_TABLE_CLASSES.EXPANDABLE}`}
+            role="cell"
+          />
+        );
+      }
+    }
+
+    if (this.config.selectable) {
+      rowCells.push(this.selectRowCheckbox(false, rowId));
+    }
+
+    let cellIndex = 0;
+
+    bodyRow.data.forEach((rowCell: any) => {
+      const columnSettings = this.data.head[
+          Object.keys(this.data.head)[cellIndex]
+        ],
+        alignment = this.cellAlignment(
+          rowCell.align ? rowCell.align : columnSettings.align || null
+        ),
+        cellLabel = rowCell.label || columnSettings.label,
+        cellWidth =
+          this.config.columnSizes && this._currentScreenBreakpoint
+            ? this.config.columnSizes[this._currentScreenBreakpoint][
+                cellIndex
+              ]
+            : null;
+      let cellData;
+
+      if (!!rowCell.template && !!this.$scopedSlots[rowCell.template]) {
+        if (typeof rowCell === 'object' && rowCell.payload) {
+          cellData = this.$scopedSlots[rowCell.template]!(rowCell.payload);
+        }
+      } else if (typeof rowCell === 'object' && !!rowCell.value) {
+        cellData = (
+          <Tooltip message={rowCell} class="-w--100">
+            <div class={UTILITY_CLASSES.TYPOGRAPHY.TEXT_TRUNCATE}>
+              {rowCell.value}
+            </div>
+          </Tooltip>
+        );
+      } else if (
+        typeof rowCell === 'string' ||
+        typeof rowCell === 'number') {
+        cellData = (
+          <Tooltip message={rowCell} class="-w--100">
+            <div class={UTILITY_CLASSES.TYPOGRAPHY.TEXT_TRUNCATE}>
+              {rowCell}
+            </div>
+          </Tooltip>
+        );
+      } else {
+        cellData = null;
+      }
+
+      rowCells.push(
+        <div
+          aria-label={cellLabel}
+          class={`
+            ${DATA_TABLE_CLASSES.CELL}
+            ${alignment}
+            ${cellWidth && cellWidth > 0 ? `-flex-basis--${cellWidth}` : ''}`}
+          data-label={cellLabel}
+          role="cell"
+          style={`
+            ${cellWidth === 0 ? 'display: none' : ''}
+            ${columnSettings.allowOverflow ? 'overflow: visible;' : ''}
+            `}
+        >
+          {cellData}
+        </div>
+      );
+
+      cellIndex++;
+    });
+
+    row.push(
+      <div
+        id={rowId}
+        class={`
+        ${rowClass}
+        ${striped ? DATA_TABLE_CLASSES.STRIPED : ''}
+        ${this.selectedRows.includes(rowId) ? ACTIVE_CLASS : ''}
+        ${this.accordionsShown.includes(rowId) ? EXPANDED_CLASS : ''}
+        `}
+        role="row"
+      >
+        {rowCells}
+      </div>
+    );
+
+    if (bodyRow.nestedContent) {
+      row.push(
+        <transition name="slide-fade">
+          <div
+            v-show={this.accordionsShown.includes(rowId)}
+            id={`row-${rowId}-content`}
```

Comment:
do we have option to configure row level tool tip message?.

Reply:
Do you mean nested row content level?
According to our UX standards, on nested row content level the text is supposed to be shown fully, without truncation and Tooltip.

![Screenshot 2021-02-05 at 18 52 08](https://user-images.githubusercontent.com/25136131/107070394-93cf3700-67e3-11eb-85c5-85c55231d377.png)

Reply:
I was pointing to if the entire row is clickable and on row hover we need to show some tool tip? (will there be any possibility of this kind of scenario?)

Reply:
I see, no, Tooltips are used on cell level only. We don't show Tooltip for the entire row